guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -47,6 +47,12 @@ Enable the {Project} Maintenance repository and the {Project} {ProjectVersion} r
 --enable {RepoRHEL8ServerSatelliteMaintenanceProductVersion} \
 --enable {RepoRHEL8ServerSatelliteServerProductVersion}
 ----
+If the {Project} {ProductVersionPrevious} system was upgraded from {RHEL} 7 to {RHEL} 8 using Leapp, manually enable the Satellite module:
```

Comment:
```suggestion
The Satellite module must be enabled manually if the {Project} {ProductVersionPrevious} system was upgraded from {RHEL} 7 to {RHEL} 8 using Leapp.
```
Another option if you like.

Reply:
Hey @mjivraja thanks, I'll restructure the sentence to: 
`Enable the Satellite module manually if the {Project} {ProductVersionPrevious} system was upgraded from {RHEL} 7 to {RHEL} 8 using Leapp.`

to avoid passive voice. What do you think?

Reply:
This even looks good :+1:

Reply:
Hey, @mjivraja I was wondering if it makes more sense to keep the sentence as is since it makes more sense to mention the condition for performing the step so that customers don't jump the gun and enable the module even when it isn't needed.

Reply:
Yes. It also makes sense. 

( I learned to prioritize the structure over sequence but need to rethink on it again.)